_rules/element-lang-matches-default-off6ek.md

```diff
@@ -0,0 +1,337 @@
+
+The second `p` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because all but a few words are English.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="fr">
+			The Dutch phrase "Hij ging met de kippen op stok" literally translates into "He went to roost with the chickens",
+			but it means that he went to bed early.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+The second `p` element has `lang` attribute value of `en` (English), which does not matches its [default language][]. The default language is Dutch because all English words are in `span` elements with a `lang` attribute value of `fr`. Both `span` elements also have an incorrect `lang` attribute in order to make sure that all targets in this example fail the rule.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			<span lang="fr">The Dutch phrase</span> "Hij ging met de kippen op stok"
+			<span lang="fr"
+				>literally translates into "He went to roost with the chickens", but it means that he went to bed early.</span
+			>
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+This `div` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `fr`. The first `p` element also has an incorrect `lang` attribute.
+
+```html
+<html lang="fr">
+	<head>
+		<title>Feu d'artifice du nouvel an</title>
+	</head>
+	<body>
+		<div lang="fr">
+			<img src="/test-assets/shared/fireworks.jpg" alt="Fireworks over Paris" />
+			<p lang="nl">
+				Bonne année !
+			</p>
+		</div>
+		<p>
+			Un feu d'artifice sera tiré depuis les jardins du Trocadéro.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+This `div` element has a `lang` attribute value of `fr` (French), which does not match its [default language][]. The default language is English because the accessible name of the `img` element is English. The `lang` attribute on the `p` element is effectively ignored. The first `p` element is not applicable because it has no [default language][] since its content is neither [visible][] nor [included in the accessibility tree][].
+
+```html
+<html lang="fr">
+	<head>
+		<title>Feu d'artifice du nouvel an</title>
+	</head>
+	<body>
+		<div lang="fr">
+			<img src="/test-assets/shared/fireworks.jpg" aria-labelledby="caption" />
+			<p lang="en" id="caption" hidden>
+				Fireworks over Paris
+			</p>
+		</div>
+		<p>
+			Un feu d'artifice sera tiré depuis les jardins du Trocadéro.
+		</p>
+	</body>
+</html>
+```
+
+### Inapplicable
```

Comment:
More inapplicable examples are needed
- one with just a lang attribute on the html element
- one with an invalid language tag